### PR TITLE
HIVE-28482: Iceberg: CTAS, CTLT query failure while fetching URI for authorization

### PR DESCRIPTION
**Pull request by SourabhBadhya**
### What changes were proposed in this pull request?

CTAS, CTLT query failure while fetching URI for authorization

### Why are the changes needed?

CTAS, CTLT query failure while fetching URI for authorization

### Does this PR introduce _any_ user-facing change?

No

### Is the change a dependency upgrade?

No

### How was this patch tested?

Unit test

### CODE REVIEW

File: ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java [hunk 1]
```diff
@@ -14375,7 +14375,11 @@ ASTNode analyzeCreateTable(
         } catch (IOException ioE) {
           LOG.debug("Exception when validate folder", ioE);
         }
+        tblLocation = location;
```
[the remark this answers is not in the document]
Comment (deniskuzZ) [reply ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java:14378]: CTLT is not part of CREATETABLE operation?

**Review by deniskuzZ — APPROVED** [review]
LGTM +1, pending tests

File: ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (SourabhBadhya) [reply ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java:14380]: Added fix and tests for CTLT

[the remark this answers is not in the document]
Comment (deniskuzZ) [reply ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java:14380]: 👍

**Pull request merged**